Bottom: 0e34443c6832af39ac93ea4c8c9757ad773e7b71
Top:    42556354c55e1f9f8af62062f5835cf41a06fdcb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-05-01 23:37:48 +0100

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 252a535..d5d7460 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -296,8 +296,8 @@ object3d_from_contours (const POLYAREA *contours,
 
       get_contour_coord_n_in_step_mm (ct, offset_in_ct, &x1, &y1);
 
-      vertices[i]           = make_vertex3d (x1, y1, COORD_TO_STEP_Z (PCB, zbot); /* Bottom */
-      vertices[npoints + i] = make_vertex3d (x1, y1, COORD_TO_STEP_Z (PCB, ztop); /* Top */
+      vertices[i]           = make_vertex3d (x1, y1, COORD_TO_STEP_Z (PCB, zbot)); /* Bottom */
+      vertices[npoints + i] = make_vertex3d (x1, y1, COORD_TO_STEP_Z (PCB, ztop)); /* Top */
 
       object3d_add_vertex (object, vertices[i]);
       object3d_add_vertex (object, vertices[npoints + i]);
